### ISSUE TITLE
Remaining Issues after PR #1

### ISSUE THREAD

**Issue by egekorkan** [thingweb/device-sandbox#2]
There are some smaller stuff remaining after #1 . I wanted to merge it to see a state of the repo. These are small issues to fix. If they are actually big, we can open separate issues.

- [ ] There is no TD for modbus, i.e. https://github.com/thingweb/device-sandbox/blob/main/modbus/calculator/calculator.tm.json is empty 
- [x] The README should list the current devices available. This should explain the calculator example and say in which protocols it is available with which programming languages and frameworks.
- [x] CoAP TM does not have `tm:ThingModel`
- [x] There should be at least links for where the schemas are coming from. Ideally, there should be either an action to sync them periodically or the current action should fetch them before testing and we never store them ourselves.
- [x] (probably a separate issue) Ideally, we should have one TM per protocol Thing, for any language and framework for that protocol. So that when we start creating the calculator in node-wot, we do not have another TM that is almost the same.
- [ ] We should have example TDs for each. Once these Things are deployed, ideally those TDs should be linked.
- [x] Node version 18 should be used.
- [x] Installation instructions are missing
